CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ After this we will switch probably to real [Semantic Versioning 2.0.0](http://se
 * Fix #797: spring-boot generator can not handle multi-profile configuration
 * Fix #1751: Build Names are suffixed with -*s2i regardless of build strategy
 * Fix #1755: Spring boot enricher does not produce a proper heath check and liveness check path when "/" is used.
+* Feature: Check maven.compiler.target property for base image detection.
 
 ### 4.3.1 (18-10-2019)
 * Updated Kubernetes client to 4.6.1
```

Comment:
@rohanKanojia this is also in italics from older PRs. How do I trim them?

Reply:
I have to apply an `*` for every bullet point

Reply:
Just do `s/-*s2i/-s2i/ `

Reply:
Oh, the -*s2i was the problem. I missed that asterik